File: dataset/human/java/file_9641.java
import java.util.Scanner;

public class Main {
	public static void main(String[] args){
		Scanner sc = new Scanner(System.in);
		int n=sc.nextInt();
		int[] d1 = new int[n];
		int[] d2 = new int[n];
		for (int i = 0; i < n; i++) {
			d1[i] = sc.nextInt();
			d2[i] = sc.nextInt();
		}
		boolean flag = false;
		for(int i=0;i<n-2;i++){
			if(d1[i]==d2[i]&&d1[i+1]==d2[i+1]&&d1[i+2]==d2[i+2]){
				flag=true;
			}
		}
		if(flag){System.out.println("Yes");}
		else{
			System.out.println("No");}
       	}
}

File: dataset/human/cpp/file_9641.cpp
#include<bits/stdc++.h>
using namespace std;

int main(){
  int N, M;
  cin >> N >> M;
  
  vector<vector<char>> vecA(N,vector<char>(N));
  for(int i=0; i<N; i++){
    for(int j=0; j<N; j++){
      cin >> vecA.at(i).at(j);
    }
  }
  
  vector<vector<char>> vecB(M,vector<char>(M));
  for(int k=0; k<M; k++){
    for(int l=0; l<M; l++){
      cin >> vecB.at(k).at(l);
    }
  }
  
  bool flag = false;
  
  for(int m=0; m<N-M+1; m++){
    for(int n=0; n<N-M+1; n++){
      int count = 0;
      if(vecA.at(m).at(n) == vecB.at(0).at(0)){
        for(int x=0; x<M; x++){
          for(int y=0; y<M; y++){
            if(vecA.at(m+x).at(n+y) == vecB.at(x).at(y)){
              count++;
            }
            if(count == M*M){
              flag = true;
            }
          }
        }
      }
    }
  }
  if(flag){
    cout << "Yes" << endl;
  }
  else{
    cout << "No" << endl;
  }
}
